<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="CustomColor" name="findresults"/>
      </item>
      <item>
       <widget class="QLineEdit" name="editfind"/>
      </item>
      <item>
       <widget class="QPushButton" name="bo">
        <property name="text">
         <string>&amp;Открыть</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonfind">
        <property name="text">
         <string>&amp;Поиск</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="id_add"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="model_add"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Year</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="year_add"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Rating</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rating_add"/>
      </item>
      <item>
       <widget class="QPushButton" name="button_add">
        <property name="text">
         <string>&amp;Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomColor</class>
   <extends>QTextBrowser</extends>
   <header location="global">customcolor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonfind</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>find()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>787</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>669</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>find()</slot>
 </slots>
</ui>
